pkg/cephmgr/cephleader.go

```diff
@@ -23,6 +23,7 @@ import (
 	etcd "github.com/coreos/etcd/client"
 	"github.com/coreos/etcd/store"
 	"github.com/rook/rook/pkg/cephmgr/client"
+	"github.com/rook/rook/pkg/cephmgr/file"
```

Comment:
i think of this package as what it takes to enable the filesystem and mds is a subset of that. Similarly, the `object` package will be what it takes to configure the object storage, which of course requires starting rgw. Are mds and rgw synonymous with file and object? They are very closely related for sure. File and object just seem to me like the higher level concept that we're orchestrating